package.json

```
@@ -2,7 +2,12 @@
   "name": "@react-native-community/geolocation",
   "version": "1.3.0",
   "description": "React Native Geolocation Module for iOS and Android",
-  "main": "js/index.js",
+  "react-native": "js/index.js",
+  "main": "web/commonjs/index.js",
```

I think we should standardise the settings we use for `bob` on the way that it has it within the [README](https://github.com/react-native-community/bob#usage) by outputting to a `lib` folder (I'm not too keen on the naming myself, I'd prefer something like `dist` or `build`, but it seems to be what others are using).

We should also add the `module` one as we're building it with `bob`.

```suggestion
  "main": "lib/commonjs/index.js",
  "module": "lib/module/index.js",
```